### ISSUE TITLE
Can't get the system to recognize a setting ...

### ISSUE THREAD

**Issue by DClassic53** [zowe/community#1789]
**Is your feature request related to a problem or limitation? Please describe.** 
I can not get the system to recognize the zowe.setup.dataset.prefix setting within the zowe.yaml file

**Describe the experience you'd like**
Can smoeone show me?

**Describe alternatives you've considered**

**Additional context**

                                                                                
MAGEE@:/u/magee@:>zwe init -c /global/zowe/zowe.yaml                            
=============================================================================== 
>> CONFIGURE ZOWE                                                               
                                                                                
------------------------------------------------------------------------------- 
>> Check if need to update runtime directory, Java and/or node.js settings in Zo
we YAML configuration                                                           
                                                                                
  * Output:                                                                     
    node: read_yaml 10: /usrlppSysplex/zowe/bin/commands/init/index.sh 42: .: zw
ecli_process_command 4: /usrlppSysplex/zowe/bin/zwe 96: FSUM7351 not found      
These configurations need to be added to your YAML configuration file:          
                                                                                
zowe:                                                                           
  runtimeDirectory: "/usrlppSysplex/zowe"               
node:                                                                           
  home: "/usrlppSysplex/nodejs/node-v12.16.1"                                   
java:                                                                           
  home: "/usr/lpp/java/current_31"                                              
                                                                                
>> Please manually update "/global/zowe/zowe.yaml" before you start Zowe.       
                                                                                
------------------------------------------------------------------------------- 
>> Initialize Zowe custom data sets                                             
                                                                                
  * Output:                                                                     
    node: read_yaml 10: /usrlppSysplex/zowe/bin/commands/init/mvs/index.sh 28: .
: zwecli_process_command 4: /usrlppSysplex/zowe/bin/zwe 96: .: zwecli_inline_exe
cute_command 9: /usrlppSysplex/zowe/bin/commands/init/index.sh 90: .: zwecli_pro
cess_command 4: /usrlppSysplex/zowe/bin/zwe 96: FSUM7351 not found              
Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined 
in Zowe YAML configuration file.                                                
MAGEE@:/u/magee@:>

**Comment by DClassic53** [comment]
This should have been opened as a bug report or a How to question ... sorry!

**Comment by DClassic53** [comment]
Select one or more files with / or action codes.                         
 EUID=1055   /global/zowe/                                                
   Type  Perm  Changed-CST6CDT   Owner      ------Size  Filename    Row 1 
 _ Dir    755  2023-01-04 10:40  BPXROOT          8192  .                 
 _ Dir    755  2022-07-05 12:05  BPXROOT          8192  ..                
 _ Dir    755  2023-01-04 10:40  BPXROOT             0  extensions        
 _ Dir    755  2020-06-11 13:36  BPXROOT          8192  instances         
 _ Dir    755  2020-06-19 09:11  ZWESVUSR         8192  keystore          
 _ Dir    755  2023-01-04 10:39  BPXROOT             0  logs              
 _ Dir    755  2023-01-04 10:39  BPXROOT             0  workspace         
 _ File   755  2023-01-04 10:44  BPXROOT         23061  zowe.yaml         
                                                                          
#-------------------------------------------------------------------------------
zowe:                                                                           
                                                                                
  #-----------------------------------------------------------------------------
  # These configurations are used by "zwe install" or "zwe init" commands.      
  #-----------------------------------------------------------------------------
  setup:                                                                        
    # MVS data set related configurations                                       
    dataset:                                                                    
      # **COMMONLY_CUSTOMIZED**                                                 
      # where Zowe MVS data sets will be installed                              
      prefix: DDS3.PDS.ZOWE                                                     
      # **COMMONLY_CUSTOMIZED**                                                 
      # PROCLIB where Zowe STCs will be copied over                             
      proclib: SYS1.PROCLIB                                                     
      # **COMMONLY_CUSTOMIZED**                                                 
      # Zowe PARMLIB

**Comment by 1000TurquoisePogs** [comment]
You are observing a shell stack trace. The first thing it shows is the thing that is wrong.
`node:... FSUM7351 not found`
node is not found.

Check on the value of node.home in the zowe.yaml
You said you have
```
node
home: "/usrlppSysplex/nodejs/node-v12.16.1"
```

Make sure the indentation is like:
```
node:
  home: "/some/path"
```

because indentation is critical to yaml.
also confirm that the value you have there is visible to the zowe STC user, and contains within it "bin/node"
in other words, when you put "/usrlppSysplex/nodejs/node-v12.16.1" you are telling zowe that "/usrlppSysplex/nodejs/node-v12.16.1/bin/node" exists. if it does not, please correct the value so that it would be the value that contains within it "bin/node".
